Assets/Mirror/Runtime/NetworkManagerHUD.cs

```diff
@@ -124,5 +126,23 @@ void OnGUI()
 
             GUILayout.EndArea();
         }
+
+        private Uri GetUri()
+        {
+            UriBuilder uriBuilder = new UriBuilder();
+
+            if (Application.platform == RuntimePlatform.WebGLPlayer)
+            {
+                uriBuilder.Scheme = "ws";
+            }
+            else
+            {
+                uriBuilder.Scheme = "tcp";
```

Comment:
what if someone uses an UDP transport on desktop?

Reply:
we could extend the NetworkManagerHUD to have a dropdown where people can specify "tcp, udp, ws",   that way we make it easy to test the different transports.

Do you have other ideas?

Reply:
Perhaps NetworkManagerHUD should evaluate the transports and self-adjust the UI for what's Available in the current context so the client player can choose which to use.  This would also demonstrate to devs how to code for same.